config/environments/test.rb

```diff
@@ -1,4 +1,16 @@
 Rails.application.configure do
+  {
+    ENGLISH_URI:  '/rest/v1/language/100/',
```

Comment:
I'm glad you both agree. I really though stuff like this would be considered controversial. I'll make a chore on piv.